_posts/en/newsletters/2018-11-06-newsletter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Bitcoin Optech Newsletter #20"
+permalink: /en/newsletters/2018/11/06/
+name: 2018-11-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains a security notice about the C
+implementation of bech32 address decoding, an analysis of a temporary
+reduction in the number of segwit blocks, a link to an interesting
+discussion about future features for LN payments, and a few notable code
+changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Bech32 security update for C implementation:** if you use the
+  [reference implementation][bech32 c] of bech32 address decoding for
+  the C programming language, you need to [update][bech32 patch] to fix
+  a potential overflow bug.  Other reference implementations are
+  unaffected; see the News section below for details.
+
+## News
+
+- **Temporary reduction in segwit block production:** Optech
+  investigated reports that a mining pool had stopped producing blocks
+  that included segwit transactions.  We found that the number of segwit
+  blocks decreased suddenly around October 20th and began rebounding
+  towards normal a few days ago.
+
+    ![Percentage of blocks including segwit transactions, last several weeks](/img/posts/segwit-blocks-2018-11.png)
+
+    A simple explanation for this sudden decrease and rebound could be a
+    minor misconfiguration.  By default, Bitcoin Core does not produce
+    segwit-including blocks in order to maintain [getblocktemplate][rpc
+    getblocktemplate] (GBT) compatibility with older pre-segwit mining
+    software.  When miners change their software or configuration, it's
+    easily possible to forget to pass the extra flag to enable segwit.
+    To illustrate how easy it is to make this mistake, the example below
+    calls GBT with its default parameter and its segwit parameter---and
+    then compares the results by the total potential block reward
+    (subsidy + fees) each block template could earn.
+
+    ```bash
+    $ ## GBT with default parameters
+    $ bitcoin-cli getblocktemplate | jq '.coinbasevalue / 1e8'
+    12.54348709
+
+    $ ## GBT with segwit enabled
+    $ bitcoin-cli getblocktemplate '{"rules": ["segwit"]}' | jq '.coinbasevalue / 1e8'
+    12.56368175
+    ```
+
+    As you can see, a miner who enabled segwit would've earned more
```

Comment:
I think the paragraph as written is fine and does not need to change, but just to point out yet another reason .02 btc is more meaningful than it may seem (as a % of 12.5), a miner will be considering their profit margin, and .02 btc is a far higher % of their profit margin than .02/12.5.